pkg/controller/vpcpeering/peering.go

```diff
@@ -545,3 +539,25 @@ func isAWSErr(err error, code string, message string) bool {
 	}
 	return false
 }
+
+func (e *external) deletePeeringByName(ctx context.Context, cr *svcapitypes.VPCPeeringConnection) error {
```

Comment:
since the argument is `*svcapitypes.VPCPeeringConnection`, I think `deleteVPCPeeringConnection` better demonstrates this function

Reply:
done.